templates/dotnet/src/Appwrite/Enums/Enums.cs.twig

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace {{ spec.title | caseUcfirst }}.Enums
+{
+    public static class {{ enum.name | caseUcfirst | overrideIdentifier }}
+    {
+        {% for value in enum.enum %}
+            {% set key = enum.keys is empty ? value : enum.keys[loop.index0] %}
+        public static string {{ key | caseUcfirst | replace({'-': ''}) }} => "{{ value }}";
+            {% if not loop.last %}{% endif %}
```

Comment:
Is this necessary here @2002Bishwajeet ?

Reply:
Not Anymore